test/functional/http/get/node.js

```diff
@@ -1,131 +1,84 @@
 'use strict';
```

Comment:
Remove the file

Reply:
Functional tests will not be removed just moved to new swagger based endpoints.